kaldb/src/main/java/com/slack/kaldb/preprocessor/PreprocessorRateLimiter.java

```diff
@@ -150,6 +151,17 @@ public Predicate<String, Trace.Span> createRateLimiter(
         return false;
       }
 
+      if (value.getTimestamp() <=0) {
```

Comment:
@bryanlb what do you think about addition validation here in the pre-processor in addition to the indexer.

If we add validation here, I think this is the code we need. But I need to go edit all unit tests to add a timestamp field .

Before I do that wanted to check if we want to add validation here :)